Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.0
+// swift-tools-version:5.7
```

Comment:
Keep 5.6 for now?

Reply:
You're right thanks https://github.com/apple/swift-log/pull/231